src/overlays/actors/ovl_En_Trt/z_en_trt.c

```diff
@@ -1053,10 +1053,7 @@ void EnTrt_ItemGiven(EnTrt* this, PlayState* play) {
                 break;
 
             default:
-                //! FAKE:
-                if (1) {}
                 this->textId = 0x849;
```

Comment:
It may be a good idea to add a comment noting this is lacking the final `break;`